Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -1278,3 +1281,25 @@ extension ComposerVC: ChatChannelControllerDelegate {
         cooldownTracker.start(with: channelController.currentCooldownTime())
     }
 }
+
+private extension Array where Element == ChatMessageAttachment<Data> {
+    var toAnyAttachmentPayload: [AnyAttachmentPayload] {
+        let decoder = JSONDecoder()
```

Comment:
Should we use our Stream's JSON decoder here? It is probably more lightweight since it uses the same Decoder instance instead of creating a new instance every time. 
```suggestion
        let decoder = JSONDecoder.stream
```

Reply:
I can see that `JSONDecoder.stream` is not accessible though it being on the LLC. Should this extension live in the LLC SDK and actually be a public API? It seems this function is important for customers wanting to support editing attachments on their custom UIs (Without using our UI SDK)

Reply:
I am not sure about it being publicly exposed 🤔 The payloads for attachments should not depend on it, do they?

Reply:
The main problem I see here is that if a customer is not using the UIKit SDK (And even SwiftUI, for this matter), it won't be able to leverage this property to support editing attachments 🤔 But let's see what @martinmitrevski thinks as well 👍